apps/client/src/common/stores/runtime.ts

```diff
@@ -38,6 +38,10 @@ export const runtimeStorePlaceholder: RuntimeStore = {
     actualStart: null,
     expectedEnd: null,
   },
+  blockState: {
```

Comment:
Our names here may not be ideal, but I am not sure `blockState` really works with the rest 

How about `currentBlock`?

Reply:
Do you have a good name for the type containing the block and extra info
maybe the type can stay `BlockState`  ?

Reply:
I think it is easier if the type name matches the object, so for `currentBlock` we would have `CurrentBlock` or `CurrentBlockState`

Reply:
I what to avoid the confusion that BurrentBlock is only the block
as there is also other data in there